runtime/wasm/effect.wat

```diff
@@ -115,20 +116,21 @@
 
    ;; Stack of fibers
 
-   (type $handlers
-      (struct
-         (field $value (ref eq))
-         (field $exn (ref eq))
-         (field $effect (ref eq))))
-
```

Comment:
Is there a particular reason for removing this type?

Reply:
Since we now update these fields instead of allocation another structure, there is no reason to keep one level of indirection.